ironfish-cli/src/commands/deposit.ts

```diff
@@ -105,11 +105,14 @@ export default class Bank extends IronfishCommand {
 
     const newBalance = oreToIron(confirmedBalance - ironToOre(IRON_TO_SEND) - fee)
 
+    const displayConfirmedBalance = displayIronAmountWithCurrency(confirmedBalance, true)
```

Comment:
This PR looks good, but there's one more thing to be wary of here -- `confirmedBalance` is in $ORE, so this will lead to incorrect output:
```
Your balance is $IRON 100,000,000.00000000 ($ORE 10,000,000,000,000,000).
You are about to send $IRON 0.10000000 ($ORE 10,000,000) plus a transaction fee of $IRON 0.00000001 ($ORE 1) to the Iron Fish deposit account.
Your remaining balance after this transaction will be $IRON 0.89999999 ($ORE 89,999,999).
The memo will contain the graffiti "test".
```

Reply:
Nice! I see your latest change just fixed this!